mmpretrain/apis/model.py

```diff
@@ -213,21 +229,103 @@ def list_models(pattern=None) -> List[str]:
         List all models:
 
         >>> from mmpretrain import list_models
-        >>> print(list_models())
+        >>> list_models()
 
         List ResNet-50 models on ImageNet-1k dataset:
 
         >>> from mmpretrain import list_models
-        >>> print(list_models('resnet*in1k'))
+        >>> list_models('resnet*in1k')
         ['resnet50_8xb32_in1k',
          'resnet50_8xb32-fp16_in1k',
          'resnet50_8xb256-rsb-a1-600e_in1k',
          'resnet50_8xb256-rsb-a2-300e_in1k',
          'resnet50_8xb256-rsb-a3-100e_in1k']
+
+        List Swin-Transformer models trained from stratch and exclude
+        Swin-Transformer-V2 models:
+
+        >>> from mmpretrain import list_models
+        >>> list_models('swin', exclude_patterns=['swinv2', '*-pre'])
+        ['swin-base_16xb64_in1k',
+         'swin-base_3rdparty_in1k',
+         'swin-base_3rdparty_in1k-384',
+         'swin-large_8xb8_cub-384px',
+         'swin-small_16xb64_in1k',
+         'swin-small_3rdparty_in1k',
+         'swin-tiny_16xb64_in1k',
+         'swin-tiny_3rdparty_in1k']
+
+        List all EVA models for image classification task.
+
+        >>> from mmpretrain import list_models
+        >>> list_models('eva', task='Image Classification')
+        ['eva-g-p14_30m-in21k-pre_3rdparty_in1k-336px',
```

Comment:
I think we need rename the old eva configs according to the new name rule

Reply:
Ok, I will do it in another PR.